src/params.cpp

```diff
@@ -162,6 +163,17 @@ void newton_param_input (const char *filenm,
             fgetsPtr = fgets(buffer,500,fp); // skip rest of line
         }
 
+        else if (strcmp(buffer,"max_numb_cells")==0) {
```

Comment:
I think we should change ``max_numb_cells`` to ``max_cells``, since ``numb`` has it's own meaning; it is just a bit confusing.